index.md

```diff
@@ -89,19 +89,55 @@ as to mitigate subjectivity in role assignment.
 
 ### [Infractions](#infractions)
 It is incumbent on members to bring issues to the attention of the community.
-An issue can't be addressed if no one else is aware it exists. Either bring up
-the issue directly with the other member or speak to some of the other members
-that you are close with. If they agree that it's an issue that needs
-addressing, the matter should be brought to the community as a whole for
-discussion.
-
-### [Sanctions](#sanctions)
-* Apologize when appropriate
-* Ameliorate the situation (e.g. repair or replace a damaged game; buy person a
-  lunch to mend things)
-
-Continued violations may result in ejection from the community, by process of
-consensus of active and long-standing members.
+An issue can't be addressed if no one else is aware it exists. Ideally, members
+should bring up the issue directly with the other member.  The next recourse is
+to initiate a formal conflict resolution proceeding.  Before doing so, it is
+highly recommended that a member speak with one or two other members about the
+matter.  If they agree that it's an issue that needs addressing, the matter
+should be brought to the attention of the Adepts and will move through the
+conflict resolution procedure described below.  Note that it is not necessary to
+relay the particulars of a conflict to the Adepts in order to begin conflict
+resolution.
+
+### Conflict Resolution
+
+#### Mediation
+Conflict resolution will initially begin with a mediation phase.  A mediator
+will be selected from the Adept group either by mutual agreement between the
```

Comment:
```suggestion
will be selected from the [Adept](#contributing-members) group either by mutual agreement between the
```

In case someone gets pointed directly to this section, making this a link will make it easier for that person to find out more about the Adept group.